molecule/default/tests/test_default.py

```diff
@@ -4,40 +4,37 @@
 import os
 
 # Third-Party Libraries
-import pytest
 import testinfra.utils.ansible_runner
 
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
     os.environ["MOLECULE_INVENTORY_FILE"]
 ).get_hosts("all")
 
 
-@pytest.mark.parametrize("pkg", ["python3-pip", "python3-dev"])
-def test_python_debian(host, pkg):
+def test_pip(host):
```

Comment:
I am not a fan of every test scenario in one test. I think it should be split on Python2 vs Python3 testing.

Reply:
The advantage of every distribution in a single test here is that it allows us to error out if we ever encounter a distribution that is not handled.  This enabled me to catch several error cases that slipped by the previous tests (Ubuntu and Kali).

Reply:
The distribution bail-out could be left in but the release check for Debian and check for Amazon Linux 2 would only live in the Python2 test.

Reply:
Please see commit 34fc121.